Wangscape/MakeReseedable.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include <noise/noise.h>
+#include "Reseedable.h"
+#include "ModuleGroup.h"
+
+void dontSetSeed(int seed);
+
+// http://stackoverflow.com/a/9531274
+// Fails in VS2015
```

Comment:
VS2017 seems to have better SFINAE support, and also seems to be freely available, so I might try it out later. Then perhaps this can be replaced with a more DRY solution.